npm/vue/package.json

```diff
@@ -30,7 +30,7 @@
     "@rollup/plugin-node-resolve": "^11.1.1",
     "@vue/cli-plugin-babel": "~4.4.0",
     "@vue/cli-service": "~4.4.0",
-    "@vue/compiler-sfc": "3.1.5",
+    "@vue/compiler-sfc": "3.2.0",
```

Comment:
I am not sure we should update deps in master prior to develop - unless the rest of the packages are already on Vue 3.2?

Reply:
agreed, I'll roll them back